openapi_tester/clients.py

```diff
@@ -3,13 +3,20 @@
 from __future__ import annotations
 
 import http
+import logging
 from typing import TYPE_CHECKING
 
-# pylint: disable=import-error
-from ninja import NinjaAPI, Router
-from ninja.testing import TestClient
+try:
+    from ninja import NinjaAPI, Router
+    from ninja.testing import TestClient
+except ImportError:
+    NinjaAPI = Router = TestClient = None
```

Comment:
This fix does not seem to work. Tested it shortly on an `ipython` session:
```
In [5]: class RestClient(TestClient):
   ...:     pass
   ...:
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-5-f427c92490b8> in <module>
----> 1 class RestClient(TestClient):
      2     pass
      3

NameError: name 'TestClient' is not defined
```

This would mean that the definition `class OpenAPINinjaClient(TestClient):`
already fails when `TestClient` evaluates to None.

I would suggest going for `NinjaAPI = Router = TestClient = object` instead, which would only create an issue as soon as `OpenAPINinjaClient` is instantiated, but not already at definition

Reply:
Thanks for spotting this, I hadn't had the time to test this properly (and that's why I hadn't it merged this yet). 
I have applied your suggested change, and updated the test. I'd appreciate if you could also give it a try on your side 👍